Src/FluentAssertions/FluentAction.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+
+namespace FluentAssertions
+{
+    /// <summary>
+    /// Contains static methods to help with exception assertions on actions.
+    /// </summary>
+    [DebuggerNonUserCode]
+    public static class FluentAction
```

Comment:
I think naming this class `FluentActions` would be better, since it is just a static container with static methods.

Reply:
Fair enough. Fixed: 0459a637bd9fc0fa93683ccced034cb440039f21